docs: document reconstruction visualization (#98)
tier4/CARET_doc: docs/visualization/visualization_api/message_flow.md

# Message flow

> To grasp the system behavior, it is important to check the message flow.

Reviewer: nits
```suggestion
To grasp the system behavior, it is useful to check the message flow.
```

> The function `message_flow()` can visualizes the latency of the target path and you can confirm which input message is used for the output message.

Reviewer: ```suggestion
The function `message_flow()` can visualizes message dependencies from path-input to path-output.
You can confirm latency of the target path.
```

```python
from caret_analyze.plot import message_flow
from caret_analyze import Application, Architecture, Lttng
from bokeh.plotting import output_notebook, figure, show
output_notebook()

arch = Architecture('yaml', '/path/to/architecture_file')
lttng = Lttng('/path/to/trace_data')
app = Application(arch, lttng)
path = app.get_path('target_path')

message_flow(path, granularity='node', lstrip_s=1, rstrip_s=1)
```

![message_flow](../../imgs/message_flow_sample.png)

The horizontal axis means `Time [s]`.
The vertical axis means the target path to measure the latency.

> Each coloerd line represents a message flow. The gray rectangular area indicates the callback execution time.

Reviewer: ```suggestion
Each coloerd line represents a message dependencies for each message. The gray rectangular area indicates the callback execution time.
```

Author: Modified.
https://github.com/tier4/CARET_doc/pull/98/commits/852b5e04f782cfa82109af7d22f6ead698c6b51a
